servant-client/test/ghcjs/package.yaml

```diff
@@ -0,0 +1,25 @@
+name: testServer
```

Comment:
I like `hpack`, but we should probably be consistent, and either switch everything to it, or nothing.